docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[session-view]]
+== Session view
+This page describes how to enable and use Session View. Session View is an investigation tool that allows you to view and analyze Linux process data organized
+in a tree according to the Linux logical event model, with processes organized by parentage and time of execution.
+It displays events in a highly readable format inspired by the terminal, making it a powerful tool for monitoring
+and investigating remote session activity on your Linux infrastructure, and understanding user and service behaviour.
+It is currently in beta.
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+1. *Interactive and non-interactive processes:* processes and services with or without a controlling tty.
+2. *User information:* the user that executed each session or process, and any exec user changes.
```

Comment:
"the user" -> "the Linux user" (just to keep things clear from challenge-response/externally-attested users in the future).